app/pages/nominees/IsIndividualNomineePreviousAddressPage.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nominees
+
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case object IsIndividualNomineePreviousAddressPage extends QuestionPage[Boolean] {
+
+  override def path: JsPath = NomineeId.path \ toString
+
+  override def toString: String = "isIndividualNomineePreviousAddress"
```

Comment:
change to `isIndividualPreviousAddress` to keep the same convention as the other individual nominee pages please